document/CoreFormat.mdk

```diff
@@ -805,83 +844,92 @@ If present, the `kind` property shall have one of the following values, with the
 * `notApplicable`: The rule specified by the `ruleId` property was not evaluated, because it does not apply to the
 file specified by `analysisTarget` (&sect;[#location-analysisTarget]).
 
-~ IsoExample
-A binary checker might have a rule that applies to 32-bit binaries only.
-Such a tool would produce a `notApplicable` result if it were run on a 64-bit binary.
+~ IsoNumberedExample { @IsoNumberedExample=1 }
+In this example, a binary checker has a rule that applies to 32-bit binaries only.
+It produces a `notApplicable` result if it is run on a 64-bit binary:
+```
+    "results": [
+        {
+            "ruleId": "ABC0001",
+            "level": "notApplicable",
+            "message": "MyTool64.exe is a 64-bit binary, so this rule does not apply to it."
```

Comment:
Can we provide a better message here rather than one that we would file a bug against if it shipped in a real tool? :)

'MyTool64.exe' was not evaluated for check ABC1001 as it is not a 32-bit binary.

or some such